src/main/g8/conf/logback.xml

```diff
@@ -23,9 +23,11 @@
     </encoder>
   </appender>
 
+  <!--
   <appender name="ASYNCFILE" class="ch.qos.logback.classic.AsyncAppender">
     <appender-ref ref="FILE" />
   </appender>
+  -->
```

Comment:
I'm wondering why the ASYNCFILE was commented out. Maybe the idea was to leave it for production only?

Reply:
For the record, the Java seed has it enabled: 
https://github.com/playframework/play-java-seed.g8/blob/2.8.x/src/main/g8/conf/logback.xml#L38

So, maybe better to align it here. By that I mean, we enable the appender in the scala seed. Users can then decided if they want it or not.

Reply:
Sorry for the delay! Fixed...